nunaliit2-js/src/main/js/nunaliit2/n2.couchNavigation.js

```diff
@@ -400,7 +412,13 @@ var NavigationService = $n2.Class({
 			var $elem = m.elem;
 			var doc = m.doc;
 			this._fixElements($elem, doc);
-		};
+		} else if ('documentContent' === m.type && m.docId === 'atlas') {
```

Comment:
I think this reads more clearly (no yoda condition). 
```suggestion
		} else if (m.type === 'documentContent'  && m.docId === 'atlas') {
```

Reply:
I don't speak yoda so must be a copy-paste error ;).  Fixed.